server/migrations/20181023192004-create-user.js

```diff
@@ -40,9 +40,9 @@ export default {
       type: Sequelize.ENUM('user', 'admin'),
       allowNull: false,
     },
-    emailConfirmation: {
-      type: Sequelize.STRING,
-      allowNull: true,
+    isConfirmed: {
```

Comment:
you should have created a new migration to modify this column ☹️

Reply:
@seyi-adeleke Oh. For every change we make we should add a new migration. Awesome. Going forward, this will be followed.